lib/src/posthog_flutter_io.dart

```diff
@@ -9,8 +11,16 @@ class PosthogFlutterIO extends PosthogFlutterPlatformInterface {
   /// The method channel used to interact with the native platform.
   final _methodChannel = const MethodChannel('posthog_flutter');
 
+  bool _isSupportedPlatform() {
+    return !(Platform.isLinux || Platform.isWindows);
+  }
```

Comment:
In the same way `isLinux` is a getter, maybe this should be a getter too?

Reply:
This should be a method since it checks different getters and returns a value based on conditions.
A getter is just an encapsulation thing that accesses a variable.